code/math/vecmat.h

```diff
@@ -21,10 +21,17 @@
 
 //Macros/functions to fill in fields of structures
 
-//macro to check if vector is zero
-#define IS_VEC_NULL(v) (fl_near_zero((v)->xyz.x) && \
-						fl_near_zero((v)->xyz.y) && \
-						fl_near_zero((v)->xyz.z))
+//macro to check if vector is zero, uses 1e-16 as a minimum tolerance
```

Comment:
Change the comment to something like this:
```
// macro to check if vector is close to zero or would be close to zero after squaring
```
And change the next comment to something like this:
```
// macro to check if vector is close to zero
```

Reply:
Also add the hash of the commit that originally split the macros into two.

Reply:
Thanks, just updated.